package capkproto;

import "capk_globals.proto";

message new_order_single {
    optional bytes		order_id = 1; 
    optional bytes		strategy_id = 2;
    optional string		symbol = 3;
	optional side_t		side = 4;
	optional double		order_qty = 5;
	optional sfixed32	ord_type = 6;
	optional double		price = 7;
	optional sfixed32	time_in_force = 8;
	optional string		account = 9;
    optional sfixed32   venue_id = 10;
};

